harlog.go

```diff
@@ -49,6 +49,13 @@ func (proxy *ProxyHttpServer) harLogAggregator() {
 			}
 
 			proxy.harLog = har.New() // reset
+		case memory := <-proxy.HarFlushToMemory:
+			proxy.Logf("Received HAR flush request to memory")
+			if len(proxy.harLog.Log.Entries) == 0 {
+				proxy.Logf("No HAR entries to flush")
+				continue
+			}
+			memory <-proxy.harLog
```

Comment:
That is true but then we will need another way to retrieve the full HAR log at some point.